<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoursePageEditor</class>
 <widget class="QWidget" name="CoursePageEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QPushButton" name="LoadFileButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="PathLabel1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь к файлу</string>
    </property>
   </widget>
   <widget class="QLabel" name="PathLabel2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C://</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddTestButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить тест</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddFileButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddDzButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить дз</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddTextButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить примечание</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddWidgetButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить компонент</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditWidgetButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить компонент</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteWidgetButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить компонент</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DoneButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить изменения</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameOnComponentLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="ComponentOrderSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="ComponentOrderLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="TutorialTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>361</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
